src/data/news/coverage.yml

```diff
@@ -1,522 +1,464 @@
-  Author: "Jake Yocom-Piatt"
   Permalink: "https://blog.decred.org/2022/12/14/Bison-Relay-The-Sovereign-Internet/"
   Params:
+    author: "Jake Yocom-Piatt"
     sortDate: "2022-12-14"
     icon: "announcementsDefault.svg"
     outlet: "Decred Blog"
-    externalSource: "blog.decred.org"
 -
   Title: "Trapped in the Web"
-  Author: "Jake Yocom-Piatt"
   Permalink: "https://blog.decred.org/2022/12/09/Trapped-in-the-Web/"
   Params:
+    author: "Jake Yocom-Piatt"
     sortDate: "2022-12-09"
     icon: "announcementsDefault.svg"
     outlet: "Decred Blog"
-    externalSource: "blog.decred.org"
 -
   Title: "What is wrong with Decred?"
-  Author: "Jake Yocom-Piatt"
   Permalink: "https://blog.decred.org/2022/12/02/What-is-wrong-with-Decred/"
   Params:
+    author: "Jake Yocom-Piatt"
     sortDate: "2022-12-02"
     icon: "announcementsDefault.svg"
     outlet: "Decred Blog"
-    externalSource: "blog.decred.org"
 -
   Title: "Jake Yocom-Piatt, Co-Founder of Decred, on Blockchain Fixing Political Voting, Campaign Spending and Privacy | Cryptonews Podcast 182"
-  Author: "Matt Zahab"
   Permalink: "https://cryptonews.com/exclusives/jake-yocom-piatt-co-founder-of-decred-on-blockchain-fixing-political-voting-campaign-spending-and-privacy.htm"
   Params:
+    author: "Matt Zahab"
     sortDate: "2022-11-29"
     icon: "interview.svg"
     outlet: "Cryptonews Podcast"
-    externalSource: "cryptonews.com"
 -
   Title: "VIDEO: Blockchain and elections | Jake Yocom-Piatt | Invezz"
-  Author: "Dan Ashmore"
   Permalink: "https://invezz.com/news/2022/11/11/video-blockchain-and-elections-jake-yocom-piatt/"
   Params:
+    author: "Dan Ashmore"
     sortDate: "2022-11-11"
     icon: "interview.svg"
     outlet: "Invezz"
-    externalSource: "invezz.com"
 -
   Title: "New Use for Blockchain: Setting the Record Straight in Political Battles"
-  Author: "Francis Bignell"
   Permalink: "https://thefintechtimes.com/new-use-for-blockchain-setting-the-record-straight-in-political-battles/"
   Params:
+    author: "Francis Bignell"
     sortDate: "2022-11-02"
     icon: "dcrInMedia.svg"
     outlet: "The Fintech Times"
-    externalSource: "thefintechtimes.com"
 -
   Title: "Jake Yocom-Piatt on Decred's PoW/PoS Hybrid And Hostile Miner Troubles"
-  Author: "Joël Valenzuela"
   Permalink: "https://www.youtube.com/watch?v=SzUtXnoJBbc"
   Params:
+    author: "Joël Valenzuela"
     sortDate: "2022-10-31"
     icon: "interview.svg"
     outlet: "Digital Cash Network"
-    externalSource: "youtube.com/@DigitalCashNetwork"
 -
   Title: "Brian ‘Buck’ Stafford of Decred On The Future of Money and Banking"
-  Author: "Jason Hartman"
   Permalink: "https://medium.com/authority-magazine/brian-buck-stafford-of-decred-on-the-future-of-money-and-banking-45f25305b409"
   Params:
+    author: "Jason Hartman"
     sortDate: "2022-09-09"
     icon: "interview.svg"
-    outlet: "Authority Magazine"
-    externalSource: "medium.com/authority-magazine"
+    outlet: "authority Magazine"
```

Comment:
Authority Magazine

Reply:
Good catch, I think that might have been a rogue find-and-replace on "Author"